docs/install-run/windows.rst

```diff
@@ -16,38 +16,37 @@ Quick Install on Windows
 Prerequisites
 =============
 
-Because CrateDB is a Java application, it runs effortlessly on Microsoft
-Windows.
+CrateDB is a Java application and runs effortlessly on Microsoft Windows.
 
-
-CrateDB versions 4.2 and above include a JVM and do not require a separate
-installation. For earlier versions,  CrateDB requires a `Java virtual machine`_
-to run. Versions from 3.0 to 4.1 require a `Java 11`_ installation. We
-recommend using `Oracle's Java`_ on Microsoft Windows. Earlier versions
-required Java 8.
-
-If you are installing on `Windows Server`_, you must install the `Microsoft
-Visual C++ 2019 Redistributable`_ package.
+If you are installing on `Windows Server`_ edition, setting up the `Microsoft
+Visual C++ 2019 Redistributable`_ package is required.
```

Comment:
Shouldn't we link directly to Microsoft?

Reply:
Thanks. I applied respected updates.